test-resources/integration-tests/start/start-happy.test.ts

```diff
@@ -1,51 +1,54 @@
 import { stackOutputs } from "../helpers/cloudformation";
-import { signedFetch } from "../helpers/fetch";
 import { kmsClient } from "../helpers/kms";
 import { JweDecrypter } from "../helpers/jwe-decrypter";
 import { JwtVerifierFactory, ClaimNames } from "../helpers/jwt-verifier";
 import { getParametersValues } from "../../headless-core-stub/utils/src/parameter/get-parameters";
-
+import { base64Encode } from "../../headless-core-stub/utils/src/base64/index";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { signedFetch } from "../helpers/fetch";
```

Comment:
[nit] it's generally good practise for imports to be in alpahateical order because it's easier to read

Reply:
I think you fixed this but it's showing out of order again?